starboard/shared/starboard/media/iamf_util.cc

```diff
@@ -0,0 +1,127 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/shared/starboard/media/iamf_util.h"
+
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "starboard/common/string.h"
+
+namespace starboard {
+namespace shared {
+namespace starboard {
+namespace media {
+
+IamfMimeUtil::IamfMimeUtil(const std::string& mime_type) {
+  // Reference: Immersive Audio Model and Formats;
+  //            v1.0.0
+  //            6.3. Codecs Parameter String
+  // (https://aomediacodec.github.io/iamf/v1.0.0-errata.html#codecsparameter)
+  if (mime_type.find("iamf") == std::string::npos) {
+    return;
+  }
```

Comment:
This is probably redundant to the check in line 57?  And if we do want to check, we should check that the return value is 0.